lib/view_component/deprecation.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require "active_support/deprecation"
+
+module ViewComponent
+  Deprecation = ActiveSupport::Deprecation.new("3.0", "ViewComponent")
```

Comment:
We will need to bump the version here when releasing v3.0, right? If yes, we could add this step to `script/release`.